### PR TITLE
Portfolio-mobile-version

### PR DESCRIPTION
**Pull request by Gh-Reza**
## Portfolio-mobile-version

# Changes:

- Added initial files
- Added Linters(Lighthouse, webint, stylelint)
- Customized README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <a href="#">
+        <img src="img/Vector.svg" alt="vector icon">
+      </a>
+      <a href="#">
+        <img src="img/twitter.svg" alt="twitter icon">
+      </a>
+      <a href="#">
+        <img src="img/m.svg" alt="m icon">
+      </a>
+    </div>
```
Comment (zieeco) [inline index.html]: - Hey, great job so far! however, to enforce good use of the html5 semantic tag, the content here is a group of list items, so you should use a `ul` as the container with an `li` to hold each `a` tags that wrap each `img`
```
<ul>
  <li><a href=""><img src="" alt=""></a></li>
</ul>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+</head>
+<body>
+  <section class="section-header flex gen-padding">
+    <a href="#" class="logo-box">
+      <img class="logo" src="img/logo.png" alt="logo">
+    </a>
+    <a href="#" class="menu-box">
+      <img class="menu-icon" src="img/menu.png" alt="menu icon">
+    </a>
+  </section>
```
Comment (zieeco) [inline index.html]: - Hey, this should be a header tag as this holds the introductory content of your webpage such as the nav items. let's make this a header tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  <title>Portfolio</title>
+</head>
+<body>
+  <section class="section-header flex gen-padding">
+    <a href="#" class="logo-box">
+      <img class="logo" src="img/logo.png" alt="logo">
+    </a>
+    <a href="#" class="menu-box">
+      <img class="menu-icon" src="img/menu.png" alt="menu icon">
+    </a>
```
Comment (zieeco) [inline index.html]: - Hi, this content should be wrapped in the nav tag as it holds the list of nav items of your webpage. please, let's wrap this content in a nav tag to improve our use of semantic tag.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  font-weight: 400;
+  width: 375px;
+  margin: 0 auto;
+}
+
+.gen-padding {
+  padding-right: 2.4rem !important;
+  padding-left: 2.4rem !important;
+}
+
```
Comment (zieeco) [inline style.css]: - Hey, according to the list of microverse [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), You should not overuse or use `!important` except you're using a CSS framework that has it applied to styles. let's try to add the styles without making use of it.

File: README.md [hunk 5]
```diff
@@ -58,231 +29,163 @@ After you're finished please remove all the comments and instructions!
...
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  cd HelloMicroverse
```
Comment (zieeco) [inline README.md]: - Hey, let's make this the current project so as not to confuse the users to improve your readme.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Gh-Reza,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Descriptive PR. ✔️
- Good commit messages :+1: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @Gh-Reza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI :heavy_check_mark: 
- Nice semantic structure :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**